Products/CMFPlone/controlpanel/browser/mail.py

```diff
@@ -37,8 +37,9 @@ def save(self):
             self.status = self.formErrorsMessage
             return False
         # keep password field
-        if (data.get('smtp_userid') is not None and
-                data.get('smtp_pass') is None):
+        smtp_user_id = (data.get('smtp_userid') or '').strip()
```

Comment:
I would fix the `smtp_user_id` vs `smtp_userid` inconsistency. But it is really minor.